app/javascript/components/RequestCart.vue

```diff
@@ -93,9 +93,6 @@
         </tbody>
       </table>
 
-    </div><!-- /.panel -->
```

Comment:
maybe keep the div with the comment and delete the other one? It's sort of helpful since it's very far away from its opening tag.